File: JavaScript/M01_JavaScriptProgrammingBasics/L02_ConditionalStatements/Exercises/P08_Scholarship.js
function scholarship(a, b, c) {
    let incomeInBgn = Number(a);
    let averageSuccess = Number(b);
    let minimalSalary = Number(c);

    let socSch = Math.floor(minimalSalary * 0.35);
    let socExcelRes = Math.floor(averageSuccess * 25);

    if (incomeInBgn < minimalSalary && averageSuccess > 4.5 && averageSuccess < 5.50) {
        console.log(`You get a Social scholarship ${socSch} BGN`)

    } else if (averageSuccess >= 5.50 && incomeInBgn < minimalSalary) {
        if (socSch > socExcelRes) {
            console.log(`You get a Social scholarship ${socSch} BGN`)
        } else {
            console.log(`You get a scholarship for excellent results ${socExcelRes} BGN`)
        }

    } else if (averageSuccess >= 5.50) {
        console.log(`You get a scholarship for excellent results ${socExcelRes} BGN`)
    } else {
        console.log(`You cannot get a scholarship!`)
    }
}

File: Python/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P08_Scholarship.py
import math
income = float(input())
average_rates = float(input())
minimum_wage = float(input())

social_money = minimum_wage * 0.35
success_money = average_rates * 25

if success_money >= social_money:
    bigger = success_money
    text = "a scholarship for excellent results"
else:
    bigger = social_money
    text = "a Social scholarship"

if income < minimum_wage and average_rates > 4.5:
    if average_rates < 5.50:   # Само социална стипендия
        print(f'You get a Social scholarship {math.floor(social_money)} BGN')
    else:                      # Социална или за успех
        print(f'You get {text} {math.floor(bigger)} BGN')
elif average_rates >= 5.5:     # Само за успех
    print(f'You get a scholarship for excellent results {math.floor(success_money)} BGN')
else:
    print('You cannot get a scholarship!')
